components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -116,7 +121,7 @@ public ActionExecutionStatus execute(ActionType actionType, Map<String, Object>
         } catch (ActionExecutionRuntimeException e) {
             DIAGNOSTIC_LOGGER.logSkippedActionExecution(actionType);
             LOG.debug("Skip executing actions for action type: " + actionType.name(), e);
-            return new ActionExecutionStatus(ActionExecutionStatus.Status.FAILED, eventContext);
+            return new ErrorStatus();
```

Comment:
Let's add an error description and a message. Let's not return empty error status objects.

Reply:
This is fixed with 148073eab6305adc3e26c71ca755a0550e805f54